hvplot/ui.py

```diff
@@ -704,6 +736,14 @@ def __init__(self, ds, **params):
             params["kind"] = "image"
         super().__init__(ds, **params)
 
+    @property
+    def _var_name(self):
+        return "ds"
```

Comment:
This could also be a `xr.DataArray`?

Reply:
Yes, I figured, it's probably more accurate to reproduce what I'm doing internally to get the explorer.

Reply:
I misread the code. You did what I expected. :upside_down_face: